bowtie/_report.py

```diff
@@ -52,6 +54,32 @@ class MissingFooter(InvalidReport):
     """
 
 
+class CaseMismatchInSameSuiteReports(Exception):
+    """
+    Two reports for the same dialect suite had some Case mismatch in them.
+    """
+
+
+class SeqResultMismatchInSameSuiteReports(Exception):
+    """
+    Two reports for the same dialect suite had some SeqResult mismatch.
```

Comment:
More specifically, what does this mean? Isn't it expected that two reports might have different test results?